scripts/ci/build.sh

```diff
@@ -2,17 +2,24 @@
 set -ex
 
 ROOTDIR="$(dirname "$0")/../.."
+PRE_RELEASE="$1"
+
 cd "${ROOTDIR}"
 
 # shellcheck disable=SC2166
 if [ "$CIRCLE_BRANCH" = release -o -n "$CIRCLE_TAG" -o -n "$FORCE_RELEASE" ]
 then
     echo -n "" >prerelease.txt
```

Comment:
I think this should report an error when `$PRE_RELEASE` is set. It's better to have it rejected than silently ignored.

Reply:
@cameel, to clarify. Do you mean throw an error when `PRE_RELEASE` is set to any value in the [scripts/ci/build.sh](https://github.com/ethereum/solidity/pull/13581/files#diff-c3f718dfc93e30493a05f25420391b061aa356fd123ea6e70171fe92004ef7d9)? If this is the case, why allow a custom pre-release string through the `PRE_RELEASE` variable then? We can just roll back this change.

Reply:
I meant only specifically in this `if` branch. The branch deals specifically with a case where we know it's not a pre-release and the custom string is ignored.

Reply:
Yeah. Exactly what I meant.